### PR TITLE
Fix #2047: Add notification for changing submission visibility to public/private

### LINKED ISSUE #2047: Display success and error message notification
## Current Behaviour
We don't show a notification for success and error messages when a participant toggles the submission visibility.

## Expected Behaviour & Deliverable
Display a notification for both cases on Success and on Error.

### PR DESCRIPTION
**Pull request by Ayukha**
Adds notification message for submission visibility update (closes #2047 )

### CODE REVIEW

**Comment by Ayukha** [comment]
@RishabhJain2018  Changes done, Please review.

**Comment by RishabhJain2018** [comment]
Can you please paste screenshot with both success and error?

File: frontend/src/js/controllers/challengeCtrl.js [hunk 1]
```diff
@@ -1005,8 +1005,15 @@
                 "is_public": vm.submissionVisibility[submission_id]
             };
             parameters.callback = {
-                onSuccess: function() {},
-                onError: function() {}
+                onSuccess: function(response) {
+                    var status = response.status;
+                    if(status === 200) {
+                      $rootScope.notify("success", "The data is successfully updated!");
```
[the remark this answers is not in the document]
Comment (Ayukha) [reply frontend/src/js/controllers/challengeCtrl.js]:
```
 { id: 4, participant_team: 1, participant_team_name: "East Diana Participant Team", execution_time: 0, challenge_phase: 1, created_by: 3, status: "finished", input_file: "http://localhost:8000/media/submission_files/submission_4/4044459d-9bfc-4fb8-b3bb-9902f2e994bb.json", stdout_file: "http://localhost:8000/media/submission_files/submission_4/f51909d8-32a9-460e-9c1d-67a1cc0f0b71.txt", stderr_file: "http://localhost:8000/media/submission_files/submission_4/6353cdc3-4fa8-447f-a378-7cdcfdf1f4b7.txt", … }
```

@RishabhJain2018 
 Update API is returning this response on success, I think we need to hardcode the success message.

[the remark this answers is not in the document]
Comment (RishabhJain2018) [reply frontend/src/js/controllers/challengeCtrl.js]: Can you please paste the compelete response?

[the remark this answers is not in the document]
Comment (Ayukha) [reply frontend/src/js/controllers/challengeCtrl.js]: Sure, Updated the response.

[the remark this answers is not in the document]
Comment (RishabhJain2018) [reply frontend/src/js/controllers/challengeCtrl.js]: I see. Can you please display the message based upon the value of `is_public` flag.
Here is the pseudo code for it. You have to write in angular :)
```
if `if_public==true`:
	message = This submission is public
else
	message = This submission is private
```

**Pull request merged**